src/olympia/promoted/admin.py

```diff
@@ -61,6 +64,23 @@ def onboarding_url(self, obj):
 
     onboarding_url.short_description = "Onboarding URL"
 
+    def stripe_information(self, obj):
+        if not obj or (obj and not obj.stripe_subscription_id):
+            return '-'
+
+        stripe_sub_url = '/'.join([
+            settings.STRIPE_DASHBOARD_URL,
+            'subscriptions',
+            obj.stripe_subscription_id
+        ])
```

Comment:
there's some extra stuff with test mode in tasks.py - do we need to take that into account here too?

Reply:
I don't think so. In `tasks.py` that's because we have an API response, which we don't have here and issuing a request for test mode seems overkill to me. In addition, I noticed our -dev Stripe account shows a page to decide whether to go to the test mode or not (in case it does not find the subscription in the default mode)